### PR TITLE
Make std modules (including merge) importable via NPM resolution

### PR DESCRIPTION
**Pull request by squaremo**
Previous changes made it possible to import e.g., `@jkcfg/std/fs` when running `jk`; but the NPM package includes things as e.g., `@jkcfg/std/std_merge`. This commit renames the modules in std/ so that they will be importable from the NPM package, the same as from the runtime. (Though not all are really expected to work with other runtimes)

### CODE REVIEW

**Comment by squaremo** [comment]
I do not understand how that unit test is failing. Firstly, eyeballing the test cases, I haven't even changed the bit that's now failing. Secondly, they pass locally. :-/

File: pkg/resolve/std_importer_test.go [hunk 1]
```diff
@@ -46,18 +46,18 @@ func TestStdImporter(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			i := &StdImporter{
 				PublicModules: []StdPublicModule{{
-					ExternalName: "std.js", InternalModule: "std.js",
+					ExternalName: "index.js", InternalModule: "index.js",
 				}, {
 					ExternalName: "param.js", InternalModule: "std_param.js",
```
Comment (dlespiau) [inline pkg/resolve/std_importer_test.go]: I think you need to change the internal module name to `param.js` here.

**Comment by dlespiau** [comment]
The issue is that the std importer will really try to import the built-in files (`module := std.Module(source)`) so the tests need to use files that have indeed been baked in

**Comment by squaremo** [comment]
Ah OK now I do understand it. There's a subtle dependency between the build, the tests, and how that importer works. Specifically: the importer needs the virtual file system to include the resolved paths; so you can't just mention any old file.

**Pull request merged**